web/server/codechecker_server/api/authentication.py

```diff
@@ -721,3 +723,102 @@ def getTokens(self):
                     str(t.last_access)))
 
             return result
+
+    @timeit
+    def newPersonalAccessToken(self, name, description):
```

Comment:
Please trim whitespace from the name of the PAT - unexpected spaces already caught me off-guard, and they don't show in the web UI.

Reply:
Done